backend/src/middlewares/validateClientMiddleware.ts

```diff
@@ -0,0 +1,16 @@
+const validateClientMiddleware = (req, res, next) => {
```

Comment:
Is this supposed to be used in other contexts than register?
If no, maybe we can rename it because we will have a `validate client` middleware for validating the client tokens

Reply:
Done !